assets/src/components/map.tsx

```diff
@@ -217,7 +212,7 @@ const LeafletShape = ({ shape }: { shape: Shape }) => {
   ])
 
   return (
-    <>
+    <div data-testid="routeShape">
```

Comment:
**question:** does wrapping this in a `<div>` change anything about the DOM such that styling or other functionality is affected? The [fragment](https://reactjs.org/docs/fragments.html) that was here previously looks kind of like a DOM node but doesn't actually create one.

Reply:
Good question, it didn't seem to based on my initial poking around, but I'd want to have more testing in place to have enough confidence in that. I'll revert to using classname testing